v1/examples/inputs/simple-input/pipeline.yaml

```diff
@@ -0,0 +1,30 @@
+pipeline:
+  name: test
+  identifier: test
+  description: description
+  projectIdentifier: hotstar
+  orgIdentifier: a_org
```

Comment:
These two field should not be present

Reply:
For now, all of my examples are with respect to V0 yaml. We wrote individual schema parsers for those only to start with. I think it would be better if I merge my sample example changes with V1 yaml spec as inputs are only applicable to them, otherwise it may be confusing for the audience. Wdyt?

Reply:
Thats cool but then you need to move these files out of v1/examples folder and put then in v0